libs/platform/src/lib/table/table.component.scss

```diff
@@ -508,13 +508,15 @@ fdk-dynamic-portal {
 
     &__body--virtual-scroll {
         background-color: var(--sapList_Background, #fff);
```

Comment:
Still lagging. I think from https://github.com/SAP/fundamental-ngx/pull/12049/files#diff-0562f5d038c3f170150e90bf1ea0967017e5e026269740c41e68a89512ead50bL507, you need also see if you can put back `position: relative;` since this resolve issue in my local.

Reply:
Issue is resolved